<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDialog</class>
 <widget class="QDialog" name="AddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>FunctionName</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="FunctionNameText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>521</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string>String</string>
   </property>
   <property name="placeholderText">
    <string>String</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="RiskLevelText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>521</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>String</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>RiskLevel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Solution</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="SolutionText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>521</width>
     <height>101</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>String</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelPushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>367</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddPushButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>367</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
